src/libPMacc/include/algorithms/math/floatMath/pow.tpp

```diff
@@ -43,6 +43,7 @@ struct Pow<float, float>
     HDINLINE result operator()(const float& base, const float& exponent)
     {
 #ifdef __CUDA_ARCH__ /*device version*/
+        /* @todo: CUDA has no pow(float, float) function yet but uses the C99 powf(float, float) */
```

Comment:
> Right. But it does not define a pow(float, float) so you have to use powf

shouldn't we just use these here / via a trait?

> Cuda 7.0 defines also a pow(float,float) and pow(float,int) but it might not be documented online.

should be good enough (?)

actually interesting, that they implement a RT version with `int` exponents...